src/test/java/org/ays/common/util/AysConfigurationProperty.java

```diff
@@ -41,33 +41,33 @@ public static class Database {
     }
 
     public static class SuperAdminUserOne {
-        public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("institution.super_admin_user_one.email_address");
-        public static final String PASSWORD = PROPERTIES.getProperty("institution.super_admin_user_one.password");
+        public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("afet_yonetim_sistemi_admin.email_address");
+        public static final String PASSWORD = PROPERTIES.getProperty("afet_yonetim_sistemi_admin.password");
     }
 
     public static class InstitutionOne {
         public static final String ID = PROPERTIES.getProperty("institution_one.id");
 
         public static class AdminUserOne {
-            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("institution_one.admin_user_one.email_address");
-            public static final String PASSWORD = PROPERTIES.getProperty("institution_one.admin_user_one.password");
+            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("volunteer_foundation_admin.email_address");
+            public static final String PASSWORD = PROPERTIES.getProperty("volunteer_foundation_admin.password");
         }
 
         public static class AdminUserTwo {
-            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("institution_one.admin_user_two.email_address");
-            public static final String PASSWORD = PROPERTIES.getProperty("institution_one.admin_user_two.password");
+            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("disaster_foundation_admin.email_address");
+            public static final String PASSWORD = PROPERTIES.getProperty("disaster_foundation_admin.password");
         }
 
         public static class TestAdmin {
-            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("institution.test_admin.email_address");
-            public static final String PASSWORD = PROPERTIES.getProperty("institution.test_admin.password");
+            public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("test_foundation_admin.email_address");
+            public static final String PASSWORD = PROPERTIES.getProperty("test_foundation_admin.password");
         }
 
     }
 
     public static class LandingUserOne {
-        public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("landing.user_one.email_address");
-        public static final String PASSWORD = PROPERTIES.getProperty("landing.user_one.password");
+        public static final String EMAIL_ADDRESS = PROPERTIES.getProperty("landing_page_user_martha.email_address");
+        public static final String PASSWORD = PROPERTIES.getProperty("landing_page_user_martha.password");
     }
```

Comment:
@cagla88 Bu sınıftaki değişken isimlerini değiştirmişiz ama class ismilerini de değiştirmemiz gerekiyor anlaşılması için

Reply:
@cagla88 Nested class'ları kaldırmayalım. Yine nested class içerisinden ulaşalım. En sağlıklı yöntem bu olur. İlerleyen dönemde farklı durumlar için kullanılmak istenirse diye böyle ilerlememiz daha iyi olabilir. Örneğin Institution'ın ID'sine ihtiyacımız olursa onu `AysConfigurationProperty.VolunteerFoundation.ID` şeklinde çağırmamız gerekecek.